### PR TITLE
Add Russian translation

### LINKED ISSUE #986: [i18n] Translate Auspice to Russian
Following the guide at:
https://github.com/nextstrain/auspice/blob/master/DEV_DOCS.md#contributing-to-internationalization-and-localization-i18nl18n

### PR DESCRIPTION
**Pull request by slavkoder**
### Description of proposed changes    
Adding a Russian translation. 

### Related issue(s)  
Fixes #986   

### Testing
What steps should be taken to test the changes you've proposed?  
- [x] Check if the Russian labels don't break formatting/css.
- [ ] Double check if the translated terms make sense in the context.

If you added or changed behavior in the codebase, did you update the tests, or do you need help with this?
- [ ] Would be great to get someone who has background in Genetics/Bio to review it.

### CODE REVIEW

**Comment by jameshadfield** [comment]
Thanks Slava! This renders well and all strings have been translated 💯 

@ivan-aksamentov would you mind double checking these translations? (or @cassiawag is there another translator of the sit-reps you could tag here?)

**Comment by cassiawag** [comment]
@frogtraveler or @irina95511, are either of you available to proofread this by reviewing the PR? Thanks!

**Comment by irina95511** [comment]
@cassiawag I will have time today in 5ish hours and happy to have a look, but I am not sure that I understand what file needs to be proofread... If there is anyone who is able to look at the file earlier, please, go ahead!

**Comment by frogtraveler** [comment]
@cassiawag  @irina95511 I also can proofread. I can start working on it around 10am PST. @cassiawag could you please send us a link to the file?

**Comment by jameshadfield** [comment]
Thanks @frogtraveler & @irina95511  -- you can see files where the russian translations of strings used in auspice has been made here: https://github.com/nextstrain/auspice/pull/995/files

File: src/locales/ru/sidebar.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "Dataset": "Данные",
+  "Date Range": "Временной Диапазон",
+  "Color By": "Цвет",
+  "Tree Options": "Настройки Дерева",
+  "Layout": "Тип диаграмы",
```
Comment (irina95511) [inline src/locales/ru/sidebar.json]: "Тип диаграммы"

File: src/locales/ru/sidebar.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "rectangular": "прямоугольный",
+  "radial": "радиальный",
+  "unrooted": "некорнеквой",
```
Comment (irina95511) [inline src/locales/ru/sidebar.json]: "некорневой"

File: src/locales/ru/sidebar.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "clock": "часовой",
+  "Branch Length": "Длинна Ветки",
```
Comment (irina95511) [inline src/locales/ru/sidebar.json]: "Длина Ветви"

File: src/locales/ru/sidebar.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "time": "время",
+  "divergence": "дивергенция",
+  "Branch Labels": "Названия Веток",
+  "Search Strains": "Поиск Штаммов",
+  "Second Tree": "Второе Дерево",
+  "Map Options": "Настройки Карты",
+  "Geographic resolution": "Географическое разделение",
+  "Animation Speed": "Скорость фнимации",
```
Comment (irina95511) [inline src/locales/ru/sidebar.json]: "Скорость Анимации"

File: src/locales/ru/sidebar.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  "Tree Options": "Настройки Дерева",
+  "Layout": "Тип диаграмы",
+  "rectangular": "прямоугольный",
+  "radial": "радиальный",
+  "unrooted": "некорнеквой",
+  "clock": "часовой",
+  "Branch Length": "Длинна Ветки",
+  "time": "время",
+  "divergence": "дивергенция",
+  "Branch Labels": "Названия Веток",
```
Comment (irina95511) [inline src/locales/ru/sidebar.json]: "Названия Ветвей"

File: src/locales/ru/translation.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  "Animation in progress.": "Идет анимация.",
+  "Filtered to": "Фильтр по",
+
+
+  "__Download Modal__": "########################################",
+  "click outside this box to return to the app": "нажмите вне поля чтобы вернуться в приложение",
+  "last updated": "последний раз обновлено",
+  "A full list of sequence authors is available via the TSV files below.": "Полный список авторов доступен в TSV файлах ниже.",
+  "Data usage policy": "Правила использования данных",
+  "Please cite the authors who contributed genomic data (where relevant), as well as:": "Пожалуйста включайте авторов, которые предоставили данные о генах (где уместно), так же как и:",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Пожалуйста, включайте авторов, которые предоставили генетические данные (где уместно), так же как и:"

File: src/locales/ru/translation.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  "Download data": "Скачать данные",
+  "Data usage part 1": "Данные представленные здесь предназанчены для быстрого распространения анализа важных патогенов. Неопубликованные данные включены с разрешения создателей и не влияет на их право публикации. ",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Данные, представленные здесь, предназначены для быстрого распространения анализа важных патогенов. Неопубликованные данные включены с разрешения создателей, что не влияет на их право публикации. "

File: src/locales/ru/translation.json [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+  "Data usage part 2": "Пожалуйста свяжитесь с соответсвующими авторами (список доступен в TSV файлах ниже) если вы собираетесь продолжать исследования используя их данные. Выводные данные, такие как филогенез, можно скачать ниже - пожалуйста свяжитесь с авторами, если уместно.",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Пожалуйста, свяжитесь с соответствующими авторами (список доступен в TSV файлах ниже), если вы собираетесь продолжать исследования используя их данные. Выводные данные, такие как филогенез, можно скачать ниже - пожалуйста, свяжитесь с авторами, если уместно."

File: src/locales/ru/translation.json [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+  "__Entropy Panel__": "########################################",
+  "Diversity": "Разновидность",
+  "entropy": "энтропия",
+  "events": "события",
+  "Codon {{codon}} in protein {{protein}}": "Кодон {{codon}} в белке {{protein}}",
+  "Nucleotide {{nuc}}": "Нуклеотид {{nuc}}",
+  "Nuc positions {{a}} to {{b}}": "Позиции нуклеоидиов {{a}} до {{b}}",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Позиции нуклеотидов {{a}} до {{b}}"

File: src/locales/ru/translation.json [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+  "Num mutations": "Кол-во мутаций",
+  "Negative strand": "Отрицательная цепь",
+  "Positive strand": "Положительная цепь",
+  "Click to color tree & map": "Нажмите чтобы заркасить дерево и карту",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Нажмите, чтобы закрасить дерево и карту"

File: src/locales/ru/translation.json [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+
+  "__Footer__": "########################################",
+  "Filter by {{filterTitle}}": "Фильтровть по {{filterTitle}}",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Фильтровать по {{filterTitle}}"

File: src/locales/ru/translation.json [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+  "Click outside this box to go back to the tree": "Нажмите за пределами поля чтобы вернуться назад к дереву",
+  "Authors": "Авторы",
+  "Title": "Название",
+  "Journal": "Журнал",
+
+
+  "__Frequencies panel__": "########################################",
+  "Frequencies": "Частоты",
+  "colored by": "окрашено по",
+  "Projection": "Проэкция",
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Проекция"

File: src/locales/ru/translation.json [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+  "Time point": "Момент времени",
+  "Frequency": "Частота",
+  "Projected frequency": "Спроэцированая частота"
```
Comment (irina95511) [inline src/locales/ru/translation.json]: "Спроецированная частота"

**Comment by irina95511** [comment]
Oh, @jameshadfield and @cassiawag it seems that I've done the wrong thing, sorry for that! I left the comments instead of changing in the file... can you see them or what should I do to make your life easier? The translation looks good to me, I just fixed some typos.

**Comment by slavkoder** [comment]
@irina95511 thanks for pointing out mistakes, I'll go and change them in the PR.

File: src/locales/ru/sidebar.json [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+  "Search Strains": "Поиск Штаммов",
+  "Second Tree": "Второе Дерево",
+  "Map Options": "Настройки Карты",
+  "Geographic resolution": "Географическое разделение",
```
Comment (frogtraveler) [inline src/locales/ru/sidebar.json]: "Географическое разрешение"

Comment (frogtraveler) [reply src/locales/ru/sidebar.json]: I can't figure out how to make changes. There is no edit option. Would it be Ok if I also add comments with proposed changes?

Comment (frogtraveler) [reply src/locales/ru/sidebar.json]: Nevermind, I found the actual file

Comment (slavkoder) [inline src/locales/ru/sidebar.json]:
```suggestion
  "Geographic resolution": "Географическое разрешение",
```

Comment (slavkoder) [reply src/locales/ru/sidebar.json]: Here is how you can make "edit suggestions":
Screen Shot 2020-03-26 at 8 41 18 AM

Then it's easy to apply changes without having to edit file.

Comment (frogtraveler) [reply src/locales/ru/sidebar.json]: Thank you! Will know next time :)

File: src/locales/ru/translation.json [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+  "__Tree (Phylogeny) panel__": "########################################",
+  "Phylogeny": "Филогенез",
+  "Reset Layout": "Сбросить Чертеж",
+  "Click on tip to display more info": "Нажмите на подсказку чтобы посмотреть детали",
+  "Click to zoom into clade": "Нажмите чтобы приблизить филогенетическую ветвь",
+  "Branch leading to": "Ветвь ведущая к",
+  "Number of descendants": "Кол-во потомков",
+  "Nucleotide mutations": "Мутации нуклеотидов",
+  "AA mutations": "Мутации Аминокислот",
+  "protein mutations truncated": "сокращенные мутации белков",
```
Comment (frogtraveler) [inline src/locales/ru/translation.json]:
```suggestion
  "protein mutations truncated": "мутации белков отображены неполностью",
```
I wasn't quite sure what is exactly displayed here. If this just means that an incomplete list of protein mutations is displayed, then my suggestion on wording is above.

**Comment by frogtraveler** [comment]
I'm done reviewing. Just had a few word choice suggestions. All looks good!
-Varvara

**Comment by jameshadfield** [comment]
Thanks for the proof reading! As my knowledge of Russian is precisely zero, @slavkoder would you mind committing any suggested changes & resolving the conversations. I'm happy to merge after this.

**Comment by slavkoder** [comment]
Done!

**Comment by jameshadfield** [comment]
Thanks, this is very cool to see more languages in auspice 😄 

One final sentence needed (this had been added to the master branch after this PR was created):
```
"Showing {{x}} of {{y}} genomes sampled between {{from}} and {{to}}."
```

**Comment by frogtraveler** [comment]
Показаны {{x}} из {{y}} собранных геномов
@jameshadfield

**Pull request merged**